config/rector.php

```diff
@@ -11,9 +16,39 @@
             __DIR__ . '/../tests',
         ]
     )
-    ->withPhpSets()
-    ->withRules(
-        [
-            // AddVoidReturnTypeWhereNoReturnRector::class,
-        ]
-    );
+    ->withPhpVersion(PhpVersion::PHP_73)
+    ->withPhpSets(
+        true
+    )
+    ->withSets([
+        // Rector sets
+
+        LevelSetList::UP_TO_PHP_73,
```

Comment:
Where are these documented?  I can't find it.

Reply:
https://github.com/rectorphp/rector/blob/main/src/Set/ValueObject/LevelSetList.php and https://github.com/rectorphp/rector/blob/main/config/set/level/up-to-php73.php etc. Basically, the `LevelSetList::UP_TO_PHP_*` set lists are about raising the PHP language level to the corresponding PHP version.